lnd_test.go

```diff
@@ -5551,40 +5551,65 @@ func testRevokedCloseRetributionRemoteHodl(net *lntest.NetworkHarness,
 	// Query the mempool for Dave's justice transaction, this should be
 	// broadcast as Carol's contract breaching transaction gets confirmed
 	// above. Since Carol might have had the time to take some of the HTLC
-	// outputs to the second level before Alice broadcasts her justice tx,
+	// outputs to the second level before Dave broadcasts his justice tx,
```

Comment:
Broadcast failure? Why's it possible for this to happen? Is it due to the new requirement of ensue the second level spends are now confirmed?

Reply:
Yes, since Dave would previously act directly on the second level spend in the mempool, this would eventually resolve. Now, with the removal of mempool spend checks, we need to mine an additional block for Dave to get notified of the second level tx.